google-cloud-core/src/main/java/com/google/cloud/GrpcServiceOptions.java

```diff
@@ -226,7 +226,7 @@ protected GrpcServiceOptions(
   /**
    * Returns a builder for API call settings.
    */
-  protected ApiCallSettings.Builder apiCallSettings() {
+  protected UnaryApiCallSettings.Builder apiCallSettings() {
```

Comment:
We have used the "paged list" terminology internally in some places. Maybe you're right, simply `PagedCallSettings` might be better. We'll discuss internally.